mysql_scripts/table_creation/FullDB.sql

```diff
@@ -189,3 +190,14 @@ CREATE TABLE UserPermission (
 	FOREIGN KEY(permissionId) REFERENCES Permission(permissionId),
 	CONSTRAINT UNIQUE unique_permission (userId, permissionId)
 );
+
+CREATE TABLE VirtualAppointmentConsent (
+	virtualAppointmentConsentId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+	createdAt DATETIME NOT NULL DEFAULT NOW(),
+	reviewConsent BOOLEAN NOT NULL,
+	virtualConsent BOOLEAN NOT NULL, /* virtual is a keyword */
+    signature VARCHAR(355) NOT NULL,
```

Comment:
Is there any reason you made this a `VARCHAR(355)` instead of `VARCHAR(255)` that (I think) we use elsewhere? It doesn't really matter, just curious

Reply:
I think I may have copied it from the first VARCHAR variable I saw: `email` in `User`, which is `VARCHAR(355)`. I changed to 255 for consistency.